.github/workflows/test.yml

```
@@ -50,7 +50,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 20.12.1
+          node-version: 20.12.2
```

```suggestion
          node-version: 20
```
バージョン固定ではなく20系最新にしておこう

---

気づかないうちにCIがバージョンが変えてくる可能性も出てくるので、細かく指定できるなら指定しておいて、更新すべきときに更新した方がセーフティかなと思うのですがどうでしょうか？

---

いやー逆で、なくべく最新に上げておいて、こけたときに早めに調査できるほうがいいなと

---

ではそうしておきますね